Khan/khan-dotfiles
# INFRA-9162: Upgrade keeper to 16.8.24

With some keys, keeper refused to return data anymore.  Upgrading to 16.8.24 fixes this issue.

(Craig indicates using keeper export still shows data, so it is a potential alternate solution.)

Issue: https://khanacademy.atlassian.net/browse/INFRA-9162

Test plan:

1. Run keeper --config ${HOME}/.keeper-config.json get L6Rn5too6B5Lm4INovoWUQ and verify there is a problem. (No data returned.)
2. Run mac-setup-keeper.sh and verify it upgrades to 16.8.24
3. Run keeper --config ${HOME}/.keeper-config.json get L6Rn5too6B5Lm4INovoWUQ and verify it returns data this time.
4. Repeat and verify that you are not asked for a password or OTP.

Additionally it is worth backing up ${HOME}/.keeper-config.json and using mac-setup-keeper.sh to regenerate this file. Rerun the last two steps of the above test plan and verify things work as expected.

Note that we are thinking to remove L6Rn5too6B5Lm4INovoWUQ. Any other keeper secret you have access to may work for this test -- although Craig was getting data for some keys under 16.5.18 so it is unclear. Though he was always seeing a warning under 16.5.18 and 16.8.24 gets rid of that warning.
